load_clean.py

```diff
@@ -20,7 +23,9 @@ def load_csv(file_path):
     return df
 
 # Call the function and store the result in the gdp_data variable
-gdp_data = load_csv("C:\\Users\\psingh\\Desktop\\Personal\\Cambridge\\Year 1\\D100\\gdp_rep0_structure\\gdp_data.csv")
+gdp_data = load_csv(r"C:/Users\psingh\Desktop\gdp_repo_structure\gdp_data.csv")
```

Comment:
The path is specific to your local machine with your username on it. It is not portable on other machines.

Reply:
Apologies, will fix that, thanks!